addons/binding/org.openhab.binding.nibeuplink/ESH-INF/i18n/nibeuplink_de.properties

```diff
@@ -0,0 +1,31 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
```

Comment:
This file contains some left over comments from the generated template (here and at the end of this file). Maybe you can clean that up

Reply:
fixed it